### PR TITLE
OCPBUGS#9159: ExternalIPs should not be used by egressIPs

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.9+

Issue: https://issues.redhat.com/browse/OCPBUGS-9159

Link to docs preview: https://57107--docspreview.netlify.app/openshift-enterprise/latest/networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-service-external-ip.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by lihongan** [comment]
@huiran0826 could you please help take a look? thanks

File: networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-service-external-ip.adoc [hunk 1]
```diff
@@ -15,6 +15,11 @@ The external network infrastructure must be configured correctly to route traffi
 
 * Your cluster is configured with ExternalIPs enabled. For more information, read xref:../../networking/configuring_ingress_cluster_traffic/configuring-externalip.adoc#configuring-externalip[Configuring ExternalIPs for services].
```
Comment (lahinson) [inline networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-service-external-ip.adoc:16]:
```suggestion
* Your cluster is configured with ExternalIPs enabled. For more information, read xref:../../networking/configuring_ingress_cluster_traffic/configuring-externalip.adoc#configuring-externalip[Configuring ExternalIPs for services].
+
[NOTE]
====
Do not use the same ExternalIP for EgressIP.
====
```

File: networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-service-external-ip.adoc [hunk 2]
```diff
@@ -15,6 +15,11 @@ The external network infrastructure must be configured correctly to route traffi
...
 
+[NOTE]
+====
+Do not use the same ExternalIP for EgressIP.
```
Comment (lahinson) [inline networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-service-external-ip.adoc:20]:
```suggestion
Do not use the same ExternalIP for the egress IP.
```

In my suggestion, I used "egress IP," but use either "egress IP" or "`EgressIP object`", depending on your intended meaning.

**Review by lahinson — COMMENTED** [review]
Looks good overall! I had 2 minor comments -- one about formatting and another about spelling.

File: networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-service-external-ip.adoc (hunk 1 above)
Comment (xenolinux) [reply networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-service-external-ip.adoc:16]: Addressed

File: networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-service-external-ip.adoc (hunk 2 above)
Comment (xenolinux) [reply networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-service-external-ip.adoc]: Addressed

**Pull request merged**